cloudability/metrics-agent
# [BETA] Azure blob upload support

#### What does this PR do?
Adds the azure blob upload support to the beta metrics-agent

#### Where should the reviewer start?

#### How should this be manually tested?
Created an image based on this build and applied the image to a metrics-agent yaml.
Set all custom azure env vars with the correct values.
Deployed that metrics-agent to a test cluster.
Confirm uploads to the Azure blob container name provided.
Tested the default upload with this new image to ensure backwards compatibility.

#### Any background context you want to provide?

#### What picture best describes this PR (optional but encouraged)?

#### What are the relevant Github Issues?

#### Developer Done List


#### For the Reviewer:

#### By approving this PR, the reviewer acknowledges that they have checked all items in this done list.

#### Reviewer/Approval Done List
